optashift-employee-rostering-shared/src/main/java/org/optaplanner/openshift/employeerostering/shared/roster/PublishResult.java

```diff
@@ -0,0 +1,37 @@
+package org.optaplanner.openshift.employeerostering.shared.roster;
+
+import java.time.LocalDate;
+
+public class PublishResult {
+    private LocalDate publishedFromInclusive;
```

Comment:
=> publishedFromDate

Reply:
Perhaps, `inclusivePublishedStart` (and `exclusivePublishEnd`)? I don't think it hurts to have the inclusive/exclusive in the name.

Reply:
It to be consistent with Shift/EmployeeAvaliability/All other date intervals fields; The inclusiveness of start and the exclusiveness of end is implict; if we were to change it here, we need to change it everywhere